spec/Cabin/SingleSeat.vspec

```diff
@@ -76,50 +74,56 @@ Height:
 
 Cushion:
```

Comment:
Maybe we should replace the term "cushion" because it's ambiguous : "Cushion" has the meaning of something which buffers ones rest either on the seating or the back like a pad or a pillow (German word: "Polster"/"Polsterung").
I think a better word would be "seating".

Reply:
No problem for me